app/models/conversation.rb

```diff
@@ -59,15 +44,12 @@ def set_read(user)
     end
   end
 
-  def public?
-    false
-  end
-
   def participant_handles
-    self.participants.map{|p| p.diaspora_handle}.join(";")
+    participants.map(&:diaspora_handle).join(";")
```

Comment:
How likely is it that we'll need the whole objects anyway if this is called? In other words could it be beneficial to `pluck` instead or are we going to load the whole objects anyway?